local-cli/eject/eject.js

```diff
@@ -84,6 +129,29 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons');
+    const hdpiPath = 'android/app/src/main/res/mipmap-hdpi/ic_launcher.png';
```

Comment:
I think it should work, we're just passing these paths to the sharp library and it should resolve it properly, forward slashes are valid for paths on windows.